ios/Exponent.xcodeproj/project.pbxproj

```diff
@@ -113,6 +113,7 @@
 		3792386F0DB44FB690751F16 /* REAIOSLogger.mm in Sources */ = {isa = PBXBuildFile; fileRef = CE32F209D83549EABA4AC64F /* REAIOSLogger.mm */; settings = {COMPILER_FLAGS = "-w"; }; };
```

Comment:
could you revert all changes to this file? there shouldn't be any changes here related to this PR.